### ISSUE TITLE
Can't find MacOS

### ISSUE THREAD

**Issue by Elvettorato** [mhussain800/Dell-Latitude-5480-Hackintosh-Monterey-EFI-i5-6th-Gen#1]
The UEFI can't find the MacOS Installer, it just says "Other OS"

**Comment by Chakradhar-0208** [comment]
> The UEFI can't find the MacOS Installer, it just says "Other OS"

Hey, did u found out any way for getting into macos

**Comment by Elvettorato** [comment]
> > The UEFI can't find the MacOS Installer, it just says "Other OS"
> 
> Hey, did u found out any way for getting into macos

No, but i contacted the creator, and we talked a bit, and he has to resolve a little issue, just wait

**Comment by mhussain800** [comment]
> The UEFI can't find the MacOS Installer, it just says "Other OS"

I have fixed the issue and uploaded the latest file. Thanks

**Comment by mhussain800** [comment]
> > > The UEFI can't find the MacOS Installer, it just says "Other OS"
> > 
> > 
> > Hey, did u found out any way for getting into macos
> 
> No, but i contacted the creator, and we talked a bit, and he has to resolve a little issue, just wait

I have fixed the issue and uploaded the latest file. Thanks

**Comment by mhussain800** [comment]
> > The UEFI can't find the MacOS Installer, it just says "Other OS"
> 
> Hey, did u found out any way for getting into macos

I have fixed the issue and uploaded the latest file. Thanks
